Templates/WebHook-Generic/index.js

```diff
@@ -1,4 +1,6 @@
-﻿module.exports = function (body, context) {
+﻿module.exports = function (context, body) {
     context.log('Webhook was triggered!');
```

Comment:
WebHooks are special cased to provide the body as the input parameter, so this is an exception to that rule.